app/views/va/VASelfEmploymentIncomeSupportSchemeView.scala.html

```diff
@@ -1,38 +0,0 @@
-@*
- * Copyright 2022 HM Revenue & Customs
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- *@
-
-@import config.AppConfig
-@import uk.gov.hmrc.webchat.client.WebChatClient
-@import views.html.templates.MainTemplate
-
-@this(
-    main_template: MainTemplate,
-    appConfig: AppConfig,
-    webChatClient: WebChatClient)
-
-@()(implicit request: Request[_], messages: play.api.i18n.Messages)
-
-@main_template(title = messages("va.global.h1.title"), navTitle = Some(messages("global.nav.va.title"))) {
-
-    <h1>@messages("va.global.h1.title")</h1>
-
-    <div class="margin-bottom">@components.webchat_container(appConfig, webChatClient)</div>
-    @components.webchat_loading_text()
-
-    <div>
-        <a class="govuk-link" id="self-employment-income-support-scheme-link" href="@appConfig.selfEmploymentIncomeSupportReturnUrl">@messages("self-employment-income-support.li1")</a>
```

Comment:
I don't think the selfEmploymentIncomeSupportReturnUrl in app.config is used anywhere else. Can you please check and remove from appconfig if required